### PR TITLE
Update & Beautify `README.md`

### LINKED ISSUE #1533: Update & Beautify `README.md`, Fix improper Tabbing
* Add `Project Title`, and move the `repo-badges` like done in *https://github.com/fossasia/open-event-api-server* :+1:
* **Fix improper Tabbing** :confounded: 
* **Fix** some improper **hyperlinks**
* **Fix** improper bullet-numbering at 6 in **Technologies used**
* Bullet up the points in **Web App Generator Test Installation**
* **Typo(s)**, If any.
*  Prettify the `README.md` :heart:

### PR DESCRIPTION
**Pull request by 0x48piraj**
### Changes:

* Added `Project Title`, and move the `repo-badges` like done in *https://github.com/fossasia/open-event-api-server* :+1:
* Fixed **improper Tabbing** in `README.md`
* Fixed an `improper-hyperlink`
* Fixed **improper numbering** at 6 in **Technologies used**
* **Bulleted** points in **Web App Generator Test Installation** :heart:
* Quoted and highlighted hyperlinks & important notes. :blush:
* Fixed few **Typo(s)** :smile:
* Beautified the `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +9,22 @@ The Open Event Web App project has two components a) an event website generator
 [![codecov](https://codecov.io/gh/fossasia/open-event-webapp/branch/development/graph/badge.svg)](https://codecov.io/gh/fossasia/open-event-webapp)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/fossasia/open-event-webapp?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+The **Open Event Web App** project has **TWO** components : 
+- **An event website generator**, *and*
```
Comment (geekyd) [inline README.md]: Please remove `, *and*`

File: README.md [hunk 2]
```diff
@@ -10,11 +9,22 @@ The Open Event Web App project has two components a) an event website generator
...
+- **The actual generated website output.** 
+
+> The web generator application can generate event websites by getting data from event `JSON` files and binary media files, that are stored in a compressed zip file. You can also access the application through a `REST API`. Websites that are generated by the "**web app generator**" can be uploaded to any web location, *e.g. on Github pages or any server (e.g. via ftp)*.
```
Comment (geekyd) [inline README.md]: Please remove `> `

File: README.md [hunk 3]
```diff
@@ -26,42 +36,53 @@ The component that is generated from the web app is the event website. Several s
...
- - [index.html](/src/www/index.html) - The webform page
- - [form.js](/src/www/js/form.js) - The script that uploads the zips, and starts the generator process
+ - **[index.html](/src/www/index.html)** - *The webform page*
+ - **[form.js](/src/www/js/form.js)** - *The script that uploads the zips, and starts the generator process*
 
-2. Generator
+**2. Generator**
 
- The generator runs on a ExpressJS server, using this main [app script](/src/app.js).
+ > **The generator runs on a ExpressJS server, using this main [app script](/src/app.js).**
```
Comment (geekyd) [inline README.md]: Lets not make this bold, and remove `> `

File: README.md [hunk 4]
```diff
@@ -26,42 +36,53 @@ The component that is generated from the web app is the event website. Several s
...
+ - **[mailer.js](/src/backend/mailer.js)** - *Sends mail to organiser notifying of successful creation*
+ - **[buildlogger.js](/src/backend/buildlogger.js)** - *Displays build logs while generating webapp*
+ - **[deploy.js](/src/backend/deploy.js)** - *Automatically deploys generated event sites to user's github account*
+ - **[gulpfile.js](/src/backend/gulpfile.js)** - *Creates minified js and css for generated webapp*
 
-4. Templates
+**4. Templates**
 
- The HTML pages of the generated website are created using Handelbars templates. You can find all the templates [here](/src/backend/templates) -
+ > The HTML pages of the generated website are created using Handelbars templates.
```
Comment (geekyd) [inline README.md]: Lets keep this simple, and remove `> `

File: README.md [hunk 5]
```diff
@@ -71,54 +92,54 @@ The webapp generator uses the following technologies -
...
+ - **[basePage.js](/src/selenium/basePage.js)** - *Contains methods common to all the pages of the event site*
+ - **[eventPage.js](/src/selenium/eventPage.js)** - *Contains methods for testing of the index page of the event*
+ - **[trackPage.js](/src/selenium/trackPage.js)** - *Contains methods for testing of the tracks page of the event*
 
 
-6. Overview Site
+**6. Overview Site**
 
-  Contains thumbnail images of sample events and main index.html file required for the [showcase site](http://sched.eventyay.com)
+  > Contains thumbnail images of sample events and main index.html file required for the [showcase site](http://sched.eventyay.com)
```
Comment (geekyd) [inline README.md]: Lets keep this simple, and remove `> `

File: README.md [hunk 6]
```diff
@@ -155,7 +176,7 @@ POST /generate
 
 All configurations are saved in the [config.js](/config.json) file.  
 
-NOTE: In this document all `config.<obj>` variables refer to the data in the *config.json* file.
+> NOTE: In this document all `config.<obj>` variables refer to the data in the *config.json* file.
```
Comment (geekyd) [inline README.md]: Lets keep this simple, and remove `> `

File: README.md [hunk 7]
```diff
@@ -178,8 +199,7 @@ We use Sendgrid to send mails, and you need a Sendgrid API to make it work. Addi
 `config.SMTP_PASSWORD` | (Falls back to config file)
 
 ### Upload
-We use AWS S3 services for storing the zip file of the generated event. These zip files are automatically deleted after 3 days from the store. Each
-uploaded file has a unique download link. `node-uuid` module is used for generating the cryptic download link of the file.
```
Comment (geekyd) [inline README.md]: Lets keep the points instead of adding a paragraph & improve the text

File: README.md (hunk 1 above)
Comment (0x48piraj) [reply README.md]: Done! ;-)

File: README.md (hunk 2 above)
Comment (0x48piraj) [reply README.md]: Done! :wink: :

File: README.md (hunk 3 above)
Comment (0x48piraj) [reply README.md]: Done, and Done :blush:

File: README.md (hunk 4 above)
Comment (0x48piraj) [reply README.md]: Done, just removed `>` because that was not **bold**

File: README.md (hunk 5 above)
Comment (0x48piraj) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (0x48piraj) [reply README.md]: Generally `NOTE` is quoted. But, Okay! Done :relaxed:

File: README.md (hunk 7 above)
Comment (0x48piraj) [reply README.md]: I did'nt got that ! Can you explain ? @geekyd

Comment (geekyd) [reply README.md]: I meant that we should keep using the points here instead of paragraph.

Comment (0x48piraj) [reply README.md]: @geekyd Can you please comment out the area, I'm not getting which point are you talking about.
I'm sorry for the inconvenience 😖

File: README.md [hunk 8]
```diff
@@ -178,8 +199,7 @@ We use Sendgrid to send mails, and you need a Sendgrid API to make it work. Addi
...
+We use **AWS S3 services** for storing the zip file of the generated event. These zip files are automatically deleted after 3 days from the store. Each uploaded file has a unique download link. `node-uuid` module is used for generating the cryptic download link of the file.
```
Comment (geekyd) [inline README.md]: @0x48piraj Please revert these changes and use points here instead of the paragraph

Comment (0x48piraj) [reply README.md]: So Sorry, I have done it! 

Thanks for your persistence :heart:

**Comment by 0x48piraj** [comment]
@geekyd 
Check it now! :+1:
